dev/public/static/js/elements-identification.js

```diff
@@ -31,24 +31,34 @@ import {
   updateStyle,
   resetStyle,
   windowAlertStatusKoNotifier,
+  getParentElementIds,
+  ShapeBpmnElementKind,
+  isChildOfSubProcess,
 } from '../../../ts/dev-bundle-index';
 
 let lastIdentifiedBpmnIds = [];
+let lastIdentifiedParentBpmnIds = [];
+let styledPoolAndLaneIds = [];
 const cssClassName = 'detection';
 let isOverlaysDisplayed = true;
 let useCSS = true;
 
-function updateStyleByAPI(bpmnIds, bpmnKind) {
-  const style = { font: { spacing: {} }, fill: {}, stroke: {}, gradient: {}, label: {} };
-
+function computeDefaultStyleByKind(bpmnKind) {
```

Comment:
**suggestion:** computeStyleUpdateByKind would be more appropriate IMHO.
This is not a "default" style, default is for the current theme, styles coming from the BPMN source.